.travis.yml

```diff
@@ -12,3 +12,5 @@ rvm:
   - jruby-19mode
   - jruby-9.1.12.0
 sudo: false
+before_script:
```

Comment:
I don't believe this `export` propagates further actually. 

Can we instead add a matrix include config that runs this explicitly instead of a conditional? You can set an `env` key with a value explicitly in travis.